src/element.ts

```diff
@@ -43,16 +42,30 @@ export abstract class Element {
     return `auto${Element.ID++}`;
   }
 
+  /**
+   * Default font for text. This is not related to music engraving. Instead, see `Flow.setMusicFont(...fontNames)`
+   * to customize the font for musical symbols placed on the score.
+   */
+  static TEXT_FONT: Required<FontInfo> = {
+    family: Font.SANS_SERIF,
+    size: Font.SIZE,
+    weight: FontWeight.NORMAL,
+    style: FontStyle.NORMAL,
+  };
+
   private context?: RenderContext;
   protected rendered: boolean;
   protected style?: ElementStyle;
   private attrs: ElementAttributes;
   protected boundingBox?: BoundingBox;
   protected registry?: Registry;
 
-  // fontStack and musicFont are both initialized by the constructor via this.setFontStack(...).
-  protected fontStack!: Font[];
-  protected musicFont!: Font;
+  /**
+   * Some elements include text.
+   * The `textFont` property contains information required to style the text (i.e., font family, size, weight, and style).
+   * It is undefined by default, and can be set using `setFont(...)` or `resetFont()`.
+   */
+  protected textFont?: Required<FontInfo>;
```

Comment:
Previously, each element implemented their own font instance variable for handling text (e.g., 10pt Arial). However, different elements handled fonts in different ways. I moved it up to Element. It's optional (because not all elements utilize text and text fonts). Elements can override the default text font specified in Element.TEXT_FONT.